packages/core/tests/oob.test.ts

```diff
@@ -767,7 +767,7 @@ describe('out of band', () => {
 
       // Use the invitation did from the first connection to create the second connection
       const outOfBandRecord2 = await faberAgent.modules.oob.createInvitation({
-        invitationDid: outOfBandRecord1.outOfBandInvitation.invitationDids[0],
+        invitationDid: faberAliceConnection.did,
```

Comment:
This is different right? InvitationDid is usually not the final did

Reply:
In this case it should be the same, since we are not setting it as an multiUse invitation. I did this trick because `OutOfBandInvitation.invitationdDids` is dynamically generating peer:2 DIDs and this does not work directly when querying peer:4 DIDs.

But I'll take a second look to it to see if everything is in place.